0-bootstrap/main.tf

```diff
@@ -162,7 +162,7 @@ module "cloudbuild_bootstrap" {
 
   terraform_apply_branches = [
     "development",
-    "non-production",
+    "non\\-production", //non-production needs a \ to ensure regex matches correct branches.
```

Comment:
In the whitepaper we used "nonprod" without a dash - Can we use "nonproduction" here to simplify the script?

Reply:
the whitepaper lists the folder names as `non-production`, or are the branches listed differently?

Reply:
In the case of Jenkins, I don't think this will change the behavior, this is to make the regex used in cloud build happy